### PR TITLE
feat(notifications): add title to the Notifications' table

### PR DESCRIPTION
**Pull request by mareklibra**
## Hey, I just made a Pull Request!

Notifications' table has newly a title with count of filtered records.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by mareklibra** [comment]
unread

---

saved

File: .changeset/khaki-pugs-perform.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-notifications': minor
+---
+
+Added title to the Notifications' table. The title clearly states the filter selectino and total count of messages. This change aligns the look and fel closer to other tables.
```
Comment (jhaals) [inline .changeset/khaki-pugs-perform.md:5]:
```suggestion
Added title to the Notifications' table. The title clearly states the filter selectin and total count of messages. This change aligns the look and feel closer to other tables.
```

File: .changeset/khaki-pugs-perform.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-notifications': minor
```
Comment (jhaals) [inline .changeset/khaki-pugs-perform.md:2]:
```suggestion
'@backstage/plugin-notifications': patch
```
Package is below 1.x so a `patch` changeset is enough for introducing new non breaking features.

**Review by jhaals — CHANGES_REQUESTED** [review]
Looking good! Minor tweaks and this should be good to go 👏

File: .changeset/khaki-pugs-perform.md (hunk 1 above)
Comment (mareklibra) [reply .changeset/khaki-pugs-perform.md]: done

**Review by jhaals — APPROVED** [review]
Great!

**Pull request merged**